e2e/periodic/input/simple.nomad

```diff
@@ -4,10 +4,12 @@ job "periodic" {
 
   constraint {
     attribute = "${attr.kernel.name}"
-    value     = "linux"
+    operator  = "set_contains_any"
+    value     = "darwin,linux"
```

Comment:
That's handy, good catch. There's probably a bunch of tests where this is safe to do. Something for a separate PR though.

Reply:
Interestingly, `set_contains_any` isn't documented in https://www.nomadproject.io/docs/job-specification/constraint - it's only documented for [affinity constraints](https://www.nomadproject.io/docs/job-specification/affinity) so we should update that too.

Also, for future readers, I considered having the constraint be based on docker os attribute - but chose not to. Windows is a bit slow, and I didn't want to run tests on LCOW Windows clients if we add some.